### PR TITLE
Sort a few plugins

### PR DESCRIPTION
**Pull request by IllusiveMan196**
Moved Main quest overhaul and Vanilla quest tweaks to appropriate-ish section.
This branch is permanent as well and serves as sorting "station" for stuff added in previous branches / pull requests.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -545,6 +563,33 @@ plugins:
...
+      - 'Morrowind - Immanuel Kant Edition.ESP'
+      - 'Oath to Saint Roris Instead.ESP'
+      - 'Pacifist Options - When it Makes Sense.ESP'
+      - 'Redoran Founder''s Helm Add On- Honor the Ancestors.ESP'
+      - 'Strange Man at Gindrala Hleran''s House Overhaul.ESP'
+      - 'Teach Nels Llendo a Lesson.ESP'
+      - 'The Corpse and the Skooma Pipe Overhaul.ESP'
+      - 'The Frostmoth Smugglers - Properly Rewarded.ESP'
+      - 'Thelas'' Pillows Overhaul.ESP'
+      - 'Therana vs Trerayna.ESP'
```
Comment (sibir-ine) [inline masterlist.yaml:602]: This should be sorted after `ROHT_2_0_8.ESP`.

**Review by sibir-ine — CHANGES_REQUESTED** [review]

File: masterlist.yaml (hunk 1 above)
Comment (IllusiveMan196) [reply masterlist.yaml:602]: Like it is now?

Comment (sibir-ine) [reply masterlist.yaml:602]: Yes. However, I'd get rid of the `Quest Overhauls` subsection & move `Main_Quest_Overhaul.ESP` back to where you had it. Subsections are usually for specific mods. If Morrowind's `Gameplay - Quests` section gets unwieldy at some point, something like `Gameplay - Quests - Overhauls` can be split off of it, but it's fine as-is at the moment.

Comment (IllusiveMan196) [reply masterlist.yaml:602]: Done.

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**